src/modules/AutoinstClass.rb

```diff
@@ -19,7 +19,10 @@ module Yast
   class AutoinstClassClass < Module
     include Yast::Logger
 
+    attr_accessor :merge_xslt_path
```

Comment:
To be honest I think it is not good idea to expose it outside and also allow to change this variable. From my POV it is constant and should be used this way. Rspec allows you to clearly overwrite constants if you need.